packages/node_modules/ciscospark/src/ciscospark.js

```diff
@@ -11,6 +11,7 @@
 require('@ciscospark/plugin-authorization');
 // explicitly load wdm, since we're relying on preDiscoveryServices and the
 // url interceptor
+require('@ciscospark/internal-plugin-mercury');
```

Comment:
@adamweeks and @mattnorris - I originally thought this was "ok" but I'm having second thoughts about including internal mercury as a default in the ciscospark package. Our sample app relies on it, but it's not possible with the currently setup to import it in the .js file there. I will make a note to revisit this.